### DISCUSSION TITLE
Using a external dll or lib from C or another language ?

### DISCUSSION THREAD

**Discussion by jmgbsas** [edubart/nelua-lang#32] [Q&A] [answered]
As the language is in alpha some important step could be the posibility to call a dll, then we can fill the holes...with require or dofile as in lua..Can nelua do that now and how? The project is amazing! I am new in lua. I hate VM for that reazon I abandon it. But in native could be great I like to do desktop aplications for now to learn,  I come from Freebasic (is compiled with gcc too), I studied and test min, rust, haxell, go, as native languages etc all are a little more complex, I don't like python, is VM too and do not have brackets  ), neither java. Thanks!

**Comment by edubart** [comment, marked as answer]
You can use C libraries with Nelua as long you make a C bindings file for the library, see how to do that in the [C interoperability section](https://nelua.io/overview/#c-interoperability) in the overview.

If you have more questions or need help you can ask in the help channel in the project's discord chat.

**Comment by jmgbsas** [comment]
Thanks!
